examples/generative/vae.py

```diff
@@ -30,10 +31,10 @@ class Sampling(layers.Layer):
 
     def call(self, inputs):
         z_mean, z_log_var = inputs
-        batch = tf.shape(z_mean)[0]
-        dim = tf.shape(z_mean)[1]
-        epsilon = tf.random.normal(shape=(batch, dim))
-        return z_mean + tf.exp(0.5 * z_log_var) * epsilon
+        batch = ops.shape(z_mean)[0]
+        dim = ops.shape(z_mean)[1]
+        epsilon = keras.random.normal(shape=(batch, dim))
```

Comment:
This needs to be seeded via `seed=self.seed_generator` (to be created in the constructor, `self.seed_generator = keras.random.SeedGenerator()`)

Reply:
I apologize! I overlooked the RNG layers' statelessness. Updated in the most recent commit.